cpp/src/backends/process/model_worker.hh

```diff
@@ -3,7 +3,15 @@
 
 #include <arpa/inet.h>
 #include <cstdio>
+
+#if __has_include(<filesystem>)
```

Comment:
Do you think we can prevent this duplicate code block in every file that includes `filesystem`? probably we can define a custom header file and include that?